### PR TITLE
Add aarch64 wheel build support

### PR DESCRIPTION
**Pull request by odidev**
Add aarch64 wheel build support.
Related to https://github.com/biopython/biopython-wheels/issues/10 @peterjc Could you please review this PR?

### CODE REVIEW

File: config.sh [hunk 1]
```diff
@@ -15,6 +15,9 @@ function run_tests {
     else
         echo "Forcing en_US.UTF-8 as workaround for encoding issues in Biopython 1.70 tests"
         # There is likely a more consise method to do this, but this works:
+	if [ "$(uname -m)" == "aarch64" ]; then
```
Comment (peterjc) [inline config.sh]: Inconsistent indentation here

**Comment by peterjc** [comment]
This is interesting - can you point me at a couple of other multibuild repositories doing this already?

(I'm aware we'll be wanting to offer wheels covering the new Apple ARM chip under macOS, and that this is already being explored)

**Comment by odidev** [comment]
> This is interesting - can you point me at a couple of other multibuild repositories doing this already?

The examples of mutlibuild repositories doing the same are - [scipy](https://github.com/MacPython/scipy-wheels) and [matplotlib](https://github.com/MacPython/matplotlib-wheels)

File: config.sh (hunk 1 above)
Comment (odidev) [reply config.sh]: > Inconsistent indentation here

Done

**Comment by peterjc** [comment]
Rerunning the single failure...

**Pull request merged**